packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -56,6 +59,47 @@ describe('Validation at REST level', () => {
     'description',
   );
 
+  context('with properties excluded from schema', () => {
+    before(() => {
+      const schema: SchemaObject = jsonToSchemaObject(
+        getJsonSchema(Product, {exclude: ['id']}),
+      );
+
```

Comment:
Can we add an assertion here for the schema with excluded properties?

Reply:
I feel that's redundant, we already have tests elsewhere to verify schema with excluded properties.

What kind of assertion do you have in mind?

Reply:
I see what you mean now! I was a bit puzzled by this too. Apparently the original tests did not account for `not` and `anyOf`:

https://github.com/strongloop/loopback-next/pull/3297/files#diff-06ce773c57624b3b68d652f875f88ebbR949-R984

I'll rework those two test to verify bigger portion of the generated schema.

Reply:
I have opened a new PR to improve the tests in the current master. Once that PR is landed, I'll update this patch, which will make the removal of `not` + `anyOf` more visible.

https://github.com/strongloop/loopback-next/pull/3906